src/pages/works/[slug].jsx

```diff
@@ -38,7 +37,7 @@ const Post = ({ post, preview }) => {
                 <AlertPreview preview={true} />
               ) : undefined}
               <PostHeader post={post} />
-              <PostBody content={post.content} />
```

Comment:
Same question about the content model in Cosmic for this slug.

Reply:
No changes required in Cosmic